.eslintrc.json

```diff
@@ -10,7 +10,7 @@
     "no-var": "error",
     "prefer-const": "error",
     "semi": ["error", "always"],
-    "space-before-function-paren": ["warn", "never"],
+    "space-before-function-paren": "off",
```

Comment:
Why this? I tried to align this rule with how most of the codebase is written. I know it is inconsistent in the codebase which is why I made it a warning